CHANGES.md

```diff
@@ -24,6 +24,18 @@ OpenSSL 3.1
 
 ### Changes between 3.0 and 3.1 [xx XXX xxxx]
 
+ * Add new function and CTRL code for TLS/SSL fingerprinting improvement.
+
+   New CTRL codes and macros (subject to change):
+
+      SSL_CTRL_GET_IANA_GROUPS - SSL_get1_iana_groups()
```

Comment:
This should be `SSL_get0_iana_groups` rather than `SSL_get1_iana_groups`.

Functions with `get1` mean the caller has to deallocate a copied/ref counted item via a call to OpenSSL.  Here, this isn't the case.  The call is returning internal memory that OpenSSL deallocates.

Reply:
renamed to SSL_get0_iana_groups